openassistant/datasets/mt_note_generation/prepare.py

```diff
@@ -0,0 +1,78 @@
+import json
+import math
+import os
+import random
+import re
+import sys
+from string import punctuation
+
+import kaggle
+import pandas as pd
+
+CLINICAL_NOTE_GENERATION_TEMPLATE = """User: Write a clinical note about a patient with the following {section}: {section_information}.
+ASSISTANT: {note}"""
+
+
+def preprocess(mt_dataset):
+    def filter_for_notes(row):
+        normalized_transcript = row["transcription"].lower()
+        if "chief complaint:" in normalized_transcript:
+            return True
+        return False
+
+    mt_dataset = mt_dataset.dropna(subset=["description", "transcription"])
+    mt_note_subset = mt_dataset[mt_dataset.apply(filter_for_notes, axis=1)]
+    return mt_note_subset
+
+
+def is_chief_complaint(section):
+    return "chief complaint" in section.lower()
+
+
+def get_conversations(dataset):
+    def normalize_transcript(x):
```

Comment:
I noticed that sometimes the output contains straggline commas or two periods. maybe we can clean that up. Should be simple. Just break up the data by "." and then remove single character sentences, etc and then join it all by "." again.

Reply:
done